Test/Tests/Integration/FloatingIP.Tests.ps1

```diff
@@ -0,0 +1,211 @@
+                    "ContrailServerNetwork",
+                    Justification="It's actually used."
+                )]
+                $ContrailServerNetwork = $MultiNode.NM.AddNetwork($null, $ServerNetwork.Name, $ServerNetwork.Subnet)
+
+                Write-Log "Creating floating IP pool: $ServerFloatingIpPoolName"
+                [Diagnostics.CodeAnalysis.SuppressMessageAttribute(
+                    "PSUseDeclaredVarsMoreThanAssignments",
+                    "ContrailFloatingIpPool",
+                    Justification="It's actually used."
+                )]
+                $ContrailFloatingIpPool = $MultiNode.NM.AddFloatingIpPool($null, $ServerNetwork.Name, $ServerFloatingIpPoolName)
+
+                $MultiNode.NM.AddPolicyToNetwork($ContrailPolicy, $ContrailClientNetwork)
+                $MultiNode.NM.AddPolicyToNetwork($ContrailPolicy, $ContrailServerNetwork)
+            }
+
+            AfterAll {
+                Write-Log "Deleting floating IP pool"
+                if (Get-Variable ContrailFloatingIpPool -ErrorAction SilentlyContinue) {
+                    $MultiNode.NM.RemoveFloatingIpPool($ContrailFloatingIpPool)
+                }
+
+                Write-Log "Deleting virtual network"
+                if (Get-Variable ContrailServerNetwork -ErrorAction SilentlyContinue) {
+                    $MultiNode.NM.RemoveNetwork($ContrailServerNetwork)
+                }
+
+                Write-Log "Deleting virtual network"
+                if (Get-Variable ContrailClientNetwork -ErrorAction SilentlyContinue) {
+                    $MultiNode.NM.RemoveNetwork($ContrailClientNetwork)
+                }
+
+                Write-Log "Deleting network policy"
+                if (Get-Variable ContrailPolicy -ErrorAction SilentlyContinue) {
+                    $MultiNode.NM.RemovePolicy($ContrailPolicy)
+                }
+            }
+
+            BeforeEach {
+                $Networks = @($ClientNetwork, $ServerNetwork)
+                foreach ($Session in $MultiNode.Sessions) {
+                    Initialize-ComputeNode -Session $Session -Networks $Networks -Configs $MultiNode.Configs
+                }
+
+                Write-Log "Creating containers"
+                Write-Log "Creating container: $ContainerClientID"
+                New-Container `
+                    -Session $MultiNode.Sessions[0] `
+                    -NetworkName $ClientNetwork.Name `
+                    -Name $ContainerClientID `
+                    -Image $ContainerImage
+
+                Write-Log "Creating containers"
+                Write-Log "Creating container: $ContainerServer1ID"
+                New-Container `
+                    -Session $MultiNode.Sessions[1] `
+                    -NetworkName $ServerNetwork.Name `
+                    -Name $ContainerServer1ID `
+                    -Image $ContainerImage
+
+                Write-Log "Creating floating IP: $ServerFloatingIpPoolName"
```

Comment:
Do we really need this message suppression?

Reply:
No. I'll remove it.